app/views/categories/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="container">
+<p style="color: green"><%= notice %></p>
+<a href="<%= url_for('/users/index') %>" class="btn light-blue darken-4 text-ligth">Back</a>
+<h2 class="center-align">Categories</h2>
+<div  id="categories">
+
+  <% @categories.each do |category| %>
+    <%= render category %>
+    <p>
+      
+    </p>
+  <% end %>
+  <button  type="submit" name="action">
+<%= link_to "Add category", new_category_path, class: "btn light-blue darken-4 text-ligth" %>
+</button>
+</div>
+
```

Comment:
[REQUIRED] Kindly reduce the height of the images on the categories page so that your design can look more like the template

![image](https://github.com/aubinleyoung/Budget-app/assets/62938735/1d603e59-140d-45d5-be80-feef0471d7be)